build-system/server/new-server/transforms/modules/test/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "esm-env": {"compiled": false},
```

Comment:
i would really prefer following the jest conventions instead where test/ would have a subdirectory with a description of the test such as `module` folder in this case and it having an `options.json`. it would provide better flexibility and familiarity as this is the convention used by jest which is a pretty well used library in the JS ecosystem

Reply:
Please take a look at the updates.